packages/squiggle-lang/__tests__/TS/Symbolic_test.ts

```diff
@@ -0,0 +1,35 @@
+import { errorValueToString } from "../../src/js/index";
+import { testRun } from "./TestHelpers";
+import * as fc from "fast-check";
+
+describe("Symbolic mean", () => {
+  let triangularInputError = {
+    tag: "Error",
+    value: {
+      tag: "RETodo",
+      value: "Triangular values must be increasing order.",
+    },
+  };
+  test("mean(triangular(x,y,z))", () => {
+    fc.assert(
+      fc.property(fc.float(), fc.float(), fc.float(), (x, y, z) => {
+        let res = testRun(`mean(triangular(${x},${y},${z}))`);
+        if (!(x < y && y < z)) {
+          expect(res).toEqual(triangularInputError);
+        } else {
+          switch (res.tag) {
+            case "Error":
+              expect(errorValueToString(res.value)).toEqual(
+                "<Test cases don't seem to be finding this>"
+              );
+            case "Ok":
+              expect(res.value).toEqual({
+                tag: "number",
+                value: (x + y + z) / 3,
```

Comment:
This seems like a trivial test to me, because this would already be done symbolically, with basically this exact line in the codebase to calculate the mean.

Reply:
Well no this is a `Jstat` subroutine so this is another instance of validating `Jstat`.

Reply:
In general I'm not bullish on this whole file, _except_ that it enforces error paths/objects.

Reply:
Ah; if it goes to Jstat though, I'm still not enthusiastic about this. 

I don't understand your last point on error paths/objects.